src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationIdentifierValidatorImpl.java

```diff
@@ -83,10 +86,16 @@ public boolean ifUserRoleExists(List<String> roles, String role) {
         return doesRoleExist;
     }
 
-    public void validateOrganisationIsActive(Organisation existingOrganisation) {
+    public void validateOrganisationIsActive(Organisation existingOrganisation, HttpStatus statusCodeToBeThrown) {
         if (OrganisationStatus.ACTIVE != existingOrganisation.getStatus()) {
-            log.error("{}:: {}", loggingComponentName, ORG_NOT_ACTIVE);
-            throw new ResourceNotFoundException(ORG_NOT_ACTIVE);
+
+            if (statusCodeToBeThrown == HttpStatus.valueOf(404)) {
```

Comment:
please use like this in both if conditions
statusCodeToBeThrown == BAD_REQUEST
statusCodeToBeThrown == NOT_FOUND

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/f66c5d305dac3836bc527ea056f143a8a6aa0771